### PR TITLE
get rid of jquery ui dialogs in app manager

### PR DESCRIPTION
**Pull request by orangejenny**
quick bit of refactoring

@nickpell 

before:
screen shot 2015-12-15 at 2 55 19 pm

after:
screen shot 2015-12-15 at 2 54 24 pm

before:
screen shot 2015-12-15 at 4 29 17 pm

after:
screen shot 2015-12-15 at 4 29 42 pm

before:
screen shot 2015-12-15 at 4 31 08 pm

after:
screen shot 2015-12-15 at 4 31 34 pm

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/partials/form_tab_advanced.html [hunk 1]
```diff
@@ -5,17 +5,28 @@
...
-                <form action="{% url "corehq.apps.app_manager.views.edit_form_attr" domain app.id form.get_unique_id 'xform' %}" method="POST" enctype="multipart/form-data">
-                    {% csrf_token %}
-                    <input type="file" id="xform_file_input" name="xform" />
-                    <input type="hidden" name="ajax" value="false" />
-                    <input type="submit" id="xform_file_submit" value="Upload" />
-                </form>
+            <div class="modal hide fade" id="upload-xform">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                    <h3>Upload XForm</h3>
```
Comment (czue) [inline corehq/apps/app_manager/templates/app_manager/partials/form_tab_advanced.html]: we should add translation markup to this page

**Comment by czue** [comment]
- looks way better? :heavy_check_mark: 
- deletes a bunch of old code? :heavy_check_mark: 
- makes front end code cleaner and more consistent? :heavy_check_mark: 

this PR gets a :100: / :100: highly recommend.

**Comment by orangejenny** [comment]
added translations, merged master

**Comment by nickpell** [comment]
kicked build

**Comment by benrudolph** [comment]
...tests were broken :flushed:... need to merge master to get them to work

**Comment by orangejenny** [comment]
merged! let's see if tests pass.

**Comment by snopoke** [comment]
This is great. So much nicer.

**Comment by orangejenny** [comment]
test passing

**Pull request merged**